azure-pipelines.yml

```diff
@@ -36,6 +36,18 @@ variables:
   modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
 
```

Comment:
@samiyaakhtar Lines 12 and 26 in this file make it appear as if AzDO won't get triggered if we modify anything under GitOps. This is probably an artifact of not wanting to run the terraform integration test for GitOps changes.

Reply:
Good find, I will update